brat/Data/Hugr.hs

```diff
@@ -164,7 +169,48 @@ hugrInt = HTOpaque "arithmetic.int.types" "int" [TANat intWidth] TBEq
 hugrFloat :: HugrType
 hugrFloat = HTOpaque "arithmetic.float.types" "float64" [] TBCopy
 
-------------------------------------- OPS --------------------------------------
+
+------------------------------------ VALUES ------------------------------------
+-----------------------  (Depends on Hugr and HugrType)  -----------------------
+
+-- Depends on `Hugr` (Maybe it should be parametric in the argument to `Hugr`?)
```

Comment:
Or, we could get rid of the argument to Hugr? Do we ever use node indices that are not `Int`?

Reply:
I think we're using the hugr `Int` parameter so we can easily sort ops based on the `Int` node Ids

Reply:
(I'll remove the comment, I don't think it needs to be parameterised)